### DISCUSSION TITLE
More Feature Suggestions/Requests

### DISCUSSION THREAD

**Discussion by CorpseEffect** [EddieDover/fvtt-player-achievements#14] [Ideas]
Hello! I wasn’t sure whether it would be better to add to the existing thread or create a new one, so I hope this is alright.

I’ve had a great time planning out achievements for my players using this module, it’s really wonderful. I am curious, though, whether there’s any possibility of a future update that would allow for some additional customization of hidden achievements. For example, adding the ability to show the name of an achievement, but not the image or flavor text. Or maybe even an option that would allow separate images for when an achievement is hidden versus revealed (thus making it possible to have a silhouette of the icon visible before the achievement has been earned)

I think these options would be fun for the players, since they’d be able to get a vague idea of what the achievements are related to and how they might earn them.

Thank you for taking the time to read this!

**Comment by EddieDover** [comment]
Issues #15 and #16 have been created.

**Comment by EddieDover** [comment]
@CorpseEffect 

Something like this for the ability to show the achievment name while cloaked? (#15 )
![image](https://github.com/EddieDover/fvtt-player-achievements/assets/464286/af970e72-55fe-4770-9145-cbccc1777d59)

![image](https://github.com/EddieDover/fvtt-player-achievements/assets/464286/59348c34-12cf-46f7-bf22-21eb9a110382)

**Comment by EddieDover** [comment]
@CorpseEffect 

And what about something like this for adding an image to show when the achievement is cloaked? (#16 )
![image](https://github.com/EddieDover/fvtt-player-achievements/assets/464286/ee9eac79-56ce-43cc-80dd-28dc95355236)

![image](https://github.com/EddieDover/fvtt-player-achievements/assets/464286/bfe8d79a-1ac6-41af-8894-5517b3487d6e)
